services/lsst-login.rst

```diff
@@ -89,9 +89,93 @@ When using an ``lsst-login`` node as a "jump host" you may also wish to configur
+   Host lsst-login03
+      HostName lsst-login03.ncsa.illinois.edu
+
+   # Define an alias and config for an internal node, which can only be reached through a login node
+   Host lsst-cnd-sub01
+      HostName lsst-condordac-sub01.ncsa.illinois.edu
+      # you may need to specify your NCSA username again
+      User ncsausername
+      # when connecting to this internal host, tunnel/jump through a login node (using an alias you defined above)
+      ProxyJump lsst-login01
+      # if you want to use your local Kerberos ticket to authenticate on the interior node, configure that:
+      GSSAPIAuthentication yes
+      PreferredAuthentications gssapi-with-mic
+      # if the internal node is a batch submit node where you might want a Kerberos ticket (e.g., to
+      # submit jobs to HTCondor), you can choose to forward your credentials:
+      GSSAPIDelegateCredentials yes
+      # if you need to configure port forwarding to the internal node, you can do that here;
+      # substitute your actual port number
+      DynamicForward yourportnumber
+
+With such config in ``~/.ssh/config`` on your local machine, your SSH connections can be significantly streamlined. Your experience may look like this:
+
+(1) Your first connection attempt involves typing your password once, on your local machine, along with a Duo push for the login node. There's no need to type your password on the login node or the internal node due to GSSAPI authentication. And your local Kerberos ticket is forwarded into your session on the internal node:
+
+.. prompt:: bash $ auto
+
+   localuser@localmachine ~ % kinit ncsauser@NCSA.EDU
+   ncsauser@NCSA.EDU's password: 
+   localuser@localmachine ~ % ssh lsst-cnd-sub01
+   Duo two-factor login for ncsauser
+   
+   Enter a passcode or select one of the following options:
+   
+    1. Duo Push to XXX-XXX-####
+   
+   Passcode or option (1-1): 1
+   Last login: Fri Aug 14 15:06:35 2020 from 141.142.181.18
+   lsst-condordac-sub01.ncsa.illinois.edu (141.142.181.231)
+     OS: CentOS 7.8.2003   HW: Dell   CPU: 24x 2.60GHz   RAM: 252 GB
+     Site: ncsa  DC: npcf  Cluster: condor_dac  Role: condor_submit
+   [ncsauser@lsst-condordac-sub01 ~]$ klist
+   Ticket cache: FILE:/tmp/krb5cc_11111_OrKJ2p97xr
+   Default principal: ncsauser@NCSA.EDU
+   
+   Valid starting       Expires              Service principal
+   08/14/2020 15:06:12  08/15/2020 01:05:59  krbtgt/NCSA.EDU@NCSA.EDU
+   [ncsauser@lsst-condordac-sub01 ~]$
+
+(2) In a 2nd terminal window, you can connect again without any need to authenticate whatsoever (thanks to your ControlMaster config):
+
+.. prompt:: bash $ auto
+
+   localuser@localmachine ~ % ssh lsst-cnd-sub01
+   Last login: Fri Aug 14 15:07:34 2020 from 141.142.181.18
+   lsst-condordac-sub01.ncsa.illinois.edu (141.142.181.231)
+     OS: CentOS 7.8.2003   HW: Dell   CPU: 24x 2.60GHz   RAM: 252 GB
+     Site: ncsa  DC: npcf  Cluster: condor_dac  Role: condor_submit
+   [ncsauser@lsst-condordac-sub01 ~]$
+
```

Comment:
May also want to show how to end a ControlMaster session:
`ssh -O exit lsst-cnd-sub01`

Reply:
Added ControlMaster exit in new commit, at the end of the recipe's demo.